src/plugins/intel_cpu/src/config.cpp

```diff
@@ -348,6 +348,15 @@ void Config::readProperties(const ov::AnyMap& prop, const ModelType modelType) {
         } else if (key == ov::hint::execution_mode.name()) {
             try {
                 executionMode = val.as<ov::hint::ExecutionMode>();
+                if (executionMode == ov::hint::ExecutionMode::ACCURACY) {
+                    // disable dynamic quantization and kv quantization for best accuracy
+                    fcDynamicQuantizationGroupSize = 0;
+                    kvCachePrecision = ov::element::f32;
+                } else {
+                    // restore to default
+                    fcDynamicQuantizationGroupSize = 32;
+                    kvCachePrecision = ov::element::u8;
```

Comment:
as I know, FP32 is also used as inference precision of ACCURACY mode is used even on Xeon with BF16.
Is it handled in a different place?

Reply:
Move changes to the location of inference precision of ACCURACY: https://github.com/openvinotoolkit/openvino/blob/5d2da3bef5de5784e9a37ab85ff43aab05a9ef79/src/plugins/intel_cpu/src/config.cpp#L381-L404